examples/MMPT/demo_sign.py

```diff
@@ -97,73 +97,57 @@ def preprocess_text(text, model_name='default'):
     return caps, cmasks
 
 
-def embed_pose(pose, model_name='default'):
```

Comment:
Could you keep these functions (and their signature)? They are imported and used for the API server (app.py).

Reply:
Good point, good advice.

Reply:
Not sure what happened to embed_pose, tbh, or why I took that out. I'll put it back in

Reply:
I suspect that you have an automatic code formatter, this is also why some double quotes are changed to single quotes.

Reply:
That, and also got messed up some copy-pasting I'm guessing